src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -830,6 +832,67 @@ public function isAbstract()
     }
 
     /**
+     * Whether this definition is deprecated, that means it should not be called
+     * anymore.
+     *
+     * @param bool   $status
+     * @param string $template Template message to use if the definition is deprecated
+     *
+     * @return Definition the current instance
+     *
+     * @throws InvalidArgumentException In case the template message contains invalid chars such as \r, \n or end of long comments.
+     * @throws InvalidArgumentException In case the template message does not contain required %service_id% variable.
```

Comment:
One line: ` @throws InvalidArgumentException When the template message is invalid.`